test/integration/model.elements.api.test.ts

```diff
@@ -105,6 +107,27 @@ describe('Registry API - retrieve Model Bpmn elements', () => {
       });
     });
 
+    test('Pass link event (with source & target) ids', () => {
+      const modelElements = bpmnElementsRegistry.getModelElementsByIds(['intermediate_catch_event_link_on_top_id', 'intermediate_throw_event_link_on_top_id']);
+
+      expect(modelElements).toHaveLength(2);
+
+      expectIntermediateCatchEvent(modelElements[0] as ShapeBpmnSemantic, {
+        id: 'intermediate_catch_event_link_on_top_id',
```

Comment:
**nitpick:** please sort properties in alphabetical order for consistency with other tests

Reply:
I already order than the other tests. I copy/paste and modify them.

Reply:
> I already order than the other tests. I copy/paste and modify them.

I don't understand this sentence 😸 It's probably missing a verb.

Reply:
Will be eventually done with #2742